week1.md

```diff
@@ -9,17 +9,172 @@ these things yet):
 
 |                        | Never heard of it | Heard of it | Played with it | Used it | Understand it internally |
 | ---------------------- | ----------------- | ----------- | -------------- | ------- | ------------------------ |
-| Node.js                |                   |             |                |         |                          |
-| Express.js             |                   |             |                |         |                          |
-| Mongo DB               |                   |             |                |         |                          |
-| Mongoose               |                   |             |                |         |                          |
-| HTTP                   |                   |             |                |         |                          |
-| REST                   |                   |             |                |         |                          |
-| Database Schemas       |                   |             |                |         |                          |
-| React                  |                   |             |                |         |                          |
-| Redux                  |                   |             |                |         |                          |
-| WebSockets             |                   |             |                |         |                          |
-| Continuous Integration |                   |             |                |         |                          |
-| git                    |                   |             |                |         |                          |
-| GitHub                 |                   |             |                |         |                          |
-| Docker                 |                   |             |                |         |                          |
+<<<<<<< HEAD
```

Comment:
Looks like you got a merge conflict here - I can help you deal with this on Thursday.